src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/chat-message-editor.ts

```diff
@@ -29,19 +31,73 @@ export class ChatMessageEditor {
         this.notifyPanel = this.editorDiv.querySelector(':scope div.post-panel .notify-call-panel');
 
         // Wiring up event listeners
+        this.editorDiv.addEventListener('click', this.editorFocusHandler);
```

Comment:
Ok... I kinda tired talking about this:
- all event handlers should have "onXxx" name pattern
- moreover, you invented another kind here: all other handlers are called "xxxListener", and you add "xxxHandler".

How can we stop this?